src/Controller/EnMarche/CitizenProjectManagerController.php

```diff
@@ -52,4 +57,52 @@ public function editAction(Request $request, CitizenProject $citizenProject, Cit
             'form_committee_support' => $this->createForm(FormType::class)->createView(),
         ]);
     }
+
+    /**
+     * @Route("/acteurs/contact", name="app_citizen_project_contact_actors")
+     * @Method("POST")
```

Comment:
Where is the get action for this form?

Reply:
There is no GET action. The first form is not a SF Form.
See https://github.com/EnMarche/en-marche.fr/blob/feature/cp-contact-actors/templates/citizen_project/list_actors.html.twig#L94

Reply:
Ok!